docs/build.md

```diff
@@ -40,6 +41,22 @@ When the controller reconciles it:
 - Validates if the container `registry` output secret exists.
 - Validates if the referenced `spec.source.url` endpoint exists.
 
+## Build Validations
+
+In order to prevent users from triggering `BuildRuns` (_execution of a Build_) that will eventually fail because of wrong or missing dependencies or configuration settings. The Build controller will validate them in advance. If all validations are successful, users can expect a `Succeeded` `Status.Reason`, however if any of the validations failed, users can rely on the `Status.Reason` and `Status.Message` fields, in order to understand the root cause.
```

Comment:
Your first sentence is not a complete sentence.  put a comma after `...or configuration settings` instead of a period, and then de-capitualize the `the` in `...the Build controller`

Reply:
you are right, thanks. Fixed!